apps/dashboard/src/model-assessment/App.tsx

```diff
@@ -89,9 +91,16 @@ export class App extends React.Component<IAppProps> {
           );
       }
     } else {
-      const staticTree = getJsonTreeAdultCensusIncome(
+      let staticTree = getJsonTreeAdultCensusIncome(
```

Comment:
If I understand correctly, you're doing "adult census" as default, and in the other two cases you load the other datasets. Why not do if three times? It just reads better.

Reply:
Yes, that is what it is doing.  Hmm, I like this style better, it's more concise.